### PR TITLE
✨ Add `add_to_version_family`

### PR DESCRIPTION
**Pull request by sunnyosun**

### CODE REVIEW

File: lamindb/dev/versioning.py [hunk 1]
```diff
@@ -75,3 +74,13 @@ def get_uid_from_old_version(
...
     return new_uid, version
+
+
+def get_new_path_from_uid(old_path: UPath, old_uid: str, new_uid: str):
+    if isinstance(old_path, LocalPathClasses):
+        # for local path, the rename target must be full path
+        new_path = old_path.as_posix().replace(old_uid, new_uid)
+    else:
+        # for cloud path, the rename target must be the last part of the path
+        new_path = old_path.name.replace(old_uid, new_uid)
```
Comment (falexwolf) [inline lamindb/dev/versioning.py:85]: Did you ensure that this in fact moves the file on S3?

**Review by falexwolf — APPROVED** [review]
Great! And should pass now!

File: lamindb/dev/versioning.py (hunk 1 above)
Comment (sunnyosun) [reply lamindb/dev/versioning.py:85]: Yes, there's a test specifically for this!

Comment (falexwolf) [reply lamindb/dev/versioning.py:85]: Okie!

**Pull request merged**